ansible/roles/compiled/templates/common/micromamba.lua

```diff
@@ -0,0 +1,23 @@
+-- -*- lua -*-
+
+local help_message = [[
+   Get micromamba working…
+]]
+
+help(help_message,"\n")
+
+whatis("Name: Micromamba")
+whatis("Category: micromamba")
+whatis("Keywords: mamba micromamba")
+whatis("Description: Base MicroMamba base environment")
+whatis("URL: https://mamba.readthedocs.io/en/latest/user_guide/micromamba.html")
+
+local mamba_dir = "{{ install_dir }}"
+local funcs = "__mamba_exe __mamba_hashr __mamba_xctivate micromamba __add_sys_prefix_to_path"
+local the_shell = os.getenv("SHELL")
+
+local home = os.getenv("HOME")
+
+
+-- Initialize mamba
+execute{cmd="source " .. mamba_dir .. "/shell_hook", modeA={"load"}}
```

Comment:
This is a really interesting lua file. I'm wondering why funcs is declared and never used.

Reply:
oh i guess its an environmental variable that is used by the shell_hook script

Reply:
very cool :)

Reply:
I got that from the anaconda lua file, which I got from elsewhere? (It's referenced in the anaconda lua file if you're interested) Have